pulp_ansible/app/models.py

```diff
@@ -178,6 +179,22 @@ class Meta:
         ]
 
 
+class CollectionVersionSignature(Content):
+    """
+    A content type representing a signature that is attached to a content unit
+    # This model might live in pulpcore if it can be abstracted.
+    """
+    TYPE = "collection_signature"
+
+    content = models.ForeignKey(CollectionVersion, on_delete=models.CASCADE, related_name="signatures")
+    public_key = models.TextField()  # How long can public keys be?
+    pubkey_fingerprint = models.TextField()
+    signing_service = models.ForeignKey(SigningService, on_delete=models.SET_NULL, related_name="signatures", null=True)
+
+    class Meta:
+        default_related_name = "%(app_label)s_%(model_name)s"
```

Comment:
If we want stages to recognize CollectionVersionSignature's as the same so it can deduplicate them, we probably should make the `content` field the uniqueness constraint.